### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by Feszp2020** [KRTirtho/spotube#1565] [Q&A]
![image_2024-06-04_11-15-19](https://github.com/KRTirtho/spotube/assets/72884391/03528b9a-f820-4128-97fc-b07a3bfb5957)
Latest version. Win 10 operating system.
At startup the design of the application floated

**Comment by Peperehobbits01** [comment]
Check that out => https://github.com/KRTirtho/spotube/issues/1553
This is a temporary fix, but I am sure it will be fixed soon.

**Reply by Feszp2020** [reply]
Thanks, that solved the problem
